extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/InsertTimeOutOfBoundsFault.java

```diff
@@ -20,10 +20,12 @@
 package org.apache.druid.msq.indexing.error;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
 import org.joda.time.Interval;
 
 import java.util.Objects;
 
+@JsonTypeName(InsertTimeOutOfBoundsFault.CODE)
```

Comment:
Do we have any unit test that tests the serialization/deserialization of this class's wire format?

Reply:
yes in org.apache.druid.msq.indexing.error.MSQFaultSerdeTest